src/corelibs/U2Script/src/UgeneContextWrapper.cpp

```diff
@@ -105,6 +109,20 @@ static void setDataSearchPaths() {
     } else if (QDir(AppContext::getWorkingDirectoryPath() + relativeDevDataDir).exists()) {
         coreLog.info("Added path: " + AppContext::getWorkingDirectoryPath() + relativeDevDataDir);
```

Comment:
User log messages should be translated. Use tr()
+ I would suggest to use lower log level (debug)

Reply:
Was this part fixed?